modin/test/exchange/dataframe_protocol/omnisci/test_protocol.py

```diff
@@ -139,6 +139,40 @@ def test_export_unaligned_at_chunks(data_has_nulls):
     df_equals(md_df, exported_df)
 
 
+@pytest.mark.parametrize("data_has_nulls", [True, False])
+def test_export_bad_chunking(data_has_nulls):
```

Comment:
```suggestion
def test_export_indivisible_chunking(data_has_nulls):
```
I mean, such chunking isn't exactly "bad", it's an edge case which we should support